src/main/resources/library/attribute-definitions.json

```diff
@@ -231,8 +231,13 @@
     "library:lmsvn": {
       "title": "Library Metadata Schema Version Number",
       "description": "Library Metadata Schema Version Number",
-      "type": "integer",
-      "default": 3,
+      "current": 3,
```

Comment:
I don't want to remove type from being an integer here but for some reason I was getting a number error on the UI when I tried to submit... tried using js/parseInt but couldn't get it to work.